website/docs/error-reference.md

```diff
@@ -844,6 +844,23 @@ class A::B < PackageSpec
 end
 ```
 
+## 3723
+
+> This error is specific to Stripe's custom `--stripe-packages` mode. If you are
+> at Stripe, please see [go/modularity](http://go/modularity) for more.
+
+The `--stripe-packages` mdoe allows packages to explicitly enumerate which other
+packages are allowed to import them by using the `visible_to` directive. If a
+package uses one or more `visible_to` lines, and is imported by a package _not_
+referenced by a `visible_to` line, then Sorbet will raise an error pointing to
```

Comment:
We generally use the word "report" instead of "raise" in docs for static errors, because `raise` can be misconstrued as translating to some sort of runtime exception, which is not the case.